aspnetcore/fundamentals/http-logging/index.md

```diff
@@ -12,88 +12,95 @@ uid: fundamentals/http-logging/index
+| `ResponseBody` |  Response <xref:Microsoft.AspNetCore.Http.HttpResponse.Body>. Logging the response body has performance implications, as it requires buffering the entire response body up to `HttpLoggingOptions.ResponseBodyLogLimit`. | 0x800 |
+| `RequestProperties` | Flag for logging a collection of HTTP Request properties, including `RequestPath`, `RequestQuery`, `RequestProtocol`, `RequestMethod`, and `RequestScheme`. | `RequestPath | RequestQuery | RequestProtocol | RequestMethod | RequestScheme` |
+| `RequestPropertiesAndHeaders` | Flag for logging HTTP Request properties and headers. Includes `RequestProperties` and `RequestHeaders`. | `RequestProperties | RequestHeaders` |
+| `ResponsePropertiesAndHeaders` | Flag for logging HTTP Response properties and headers. Includes `ResponseStatusCode` and `ResponseHeaders`. | `ResponseStatusCode | ResponseHeaders` |
+| `Request` | Flag for logging the entire HTTP Request. Includes `RequestPropertiesAndHeaders` and `RequestBody`. Logging the request body has performance implications, as it requires buffering the entire request body up to `HttpLoggingOptions.RequestBodyLogLimit`. | `RequestPropertiesAndHeaders | RequestBody` |
+| `Response` | Flag for logging the entire HTTP Response. Includes `ResponsePropertiesAndHeaders` and `ResponseBody`. Logging the response body has performance implications, as it requires buffering the entire response body up to `HttpLoggingOptions.ResponseBodyLogLimit`. | `ResponseStatusCode | ResponseHeaders | ResponseBody` |
+| `All` | Flag for logging both the HTTP Request and Response. Includes `Request` and `Response`. Logging the request and response body has performance implications, as it requires buffering the entire request and response body up to the `HttpLoggingOptions.RequestBodyLogLimit` and `HttpLoggingOptions.ResponseBodyLogLimit`. | `Request | Response` |
+
+### `RequestHeaders`
+
+`RequestHeaders` are a set of HTTP Request Headers that are allowed to be logged. Header values are only logged for header names that are in this collection.
 
 [!code-csharp[](samples/6.x/Startup.cs?name=configureservices&highlight=7)]
 
-### ResponseHeaders
+### `ResponseHeaders`
 
-ResponseHeaders are a set of HTTP Request Headers that are allowed to be logged. Header values will only be logged for header names that are in this collection.
+`ResponseHeaders` are a set of HTTP Response Headers that are allowed to be logged. Header values are only logged for header names that are in this collection.
 
 [!code-csharp[](samples/6.x/Startup.cs?name=configureservices&highlight=8)]
 
-### MediaTypeOptions
+### `MediaTypeOptions`
 
-MediaTypeOptions provides configuration for selecting which encoding to use for a specific media type. 
+`MediaTypeOptions` provides configuration for selecting which encoding to use for a specific media type. 
```

Comment:
I think I forgot to expand on this type: https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/HttpLogging/src/MediaTypeOptions.cs. Could you follow a similar model to what we did prior with options?

Reply:
@serpent5 can you point out what options need documenting?

Reply:
The entire class need documentation (isn't a lot)
AddText and overloads
AddBinary and overloads
Clear